examples/apps/ai-collab/package.json

```diff
@@ -26,9 +26,7 @@
 		"format:biome": "biome check . --write",
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
-		"next:build": "next build",
-		"next:dev": "next dev",
-		"next:start": "next start",
```

Comment:
Oh? We will not start the Next.js application in production mode?

Reply:
No; the workflow we've been using just starts it in dev mode, which is good enough for the sample app.